### PR TITLE
MAT-8277 add display ids for groups and populations

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR Template

Jira Ticket: [MAT-8277](https://jira.cms.gov/browse/MAT-8277)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included.
* [x] No extemporaneous files are included (i.e. Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/models/measure/Group.java [hunk 1]
```diff
@@ -55,4 +55,6 @@ public class Group implements GroupScoringPopulation {
 
   @Min(1)
   private Integer scoringPrecision;
+
+  private String displayId;
```
Comment (adongare) [inline src/main/java/gov/cms/madie/models/measure/Group.java:59]: we might need to add validator as well or is it going to be new ticket?

Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/models/measure/Group.java:59]: @adongare The validator is in measure-service, when group is added or updated.

**Review by adongare — APPROVED** [review]
